definitions/npm/query-string_v5.1.x/flow_v0.32.x-/query-string_v5.1.x.js

```diff
@@ -11,9 +11,9 @@ declare module 'query-string' {
   |}
 
   declare module.exports: {
-    extract(str: string): string,
-    parse(str: string, opts?: ParseOptions): Object,
-    parseUrl(str: string, opts?: ParseOptions): { url: string, query: Object },
-    stringify(obj: Object, opts?: StringifyOptions): string,
+    extract(input: string): string,
+    parse(input: string, options?: ParserOptions): Object,
```

Comment:
We can type the result better. `{ [string]: string }`

Reply:
👍  Good catch, although this should also be `{ [string]: mixed }` if I'm correct: https://github.com/sindresorhus/query-string/blob/master/index.js#L153

I'm making the changes.

Reply:
Or would `{ [string]: string | Array<string> }` be better?

Reply:
You're right, it could be an array. So yes `{ [string]: string | Array<string> }` I think would be best.